README.md

```diff
@@ -95,10 +95,13 @@ _Note: you need to recompile your project after adding new fonts, also ensure th
 
 If you want to use the TabBar/NavigatorIOS integration or use `getImageSource`, then you need to add `RNVectorIcons.xcodeproj` to **Libraries** and add `libRNVectorIcons.a` to **Link Binary With Libraries** under **Build Phases**. [More info and screenshots about how to do this is available in the React Native documentation](http://facebook.github.io/react-native/docs/linking-libraries-ios.html#content).
 
+
 #### Option: With `react-native link`
 
 `$ react-native link react-native-vector-icons`
 
+Note: If you are using react-native version 0.60 or higher you don't need to link [react-native-vector-icons](https://github.com/oblador/react-native-vector-icons)
```

Comment:
Even though autolinking works fine, there is still extra steps needed to make the fonts available. There are many issues in this repo pointint out to this with different workarounds. It would be great to hear the stance of maintainers on this.

What I personally did was to rely on autolinking with the workarounds mentioned here: #1105